### PR TITLE
Wreck the wastes

### PR DESCRIPTION
**Pull request by OdysiaHallow**
Wrecks several several buildings, including the school (pic below). Fixed a lot of mapping bugs including a space tile in the sewers. Removed the motel near the old khan base, replaced it with desert to help compensate for the lack of desert tiles.
image

### CODE REVIEW

File: _maps/map_files/Pahrump/Pahrump-Surface-2.dmm [hunk 1]
```diff
@@ -24336,6 +24376,19 @@
...
+/obj/structure/destructible/tribal_torch/wall/lit{
+	dir = 1;
+	pixel_y = 23
+	},
+/turf/open/floor/f13{
+	color = "#a6968e";
+	icon_state = "darkrusty";
+	name = "dirty floor"
+	},
+/area/f13/wasteland{
```
Comment (MarinaGryphon) [inline _maps/map_files/Pahrump/Pahrump-Surface-2.dmm]:
```suggestion
/area/f13/wasteland)
```

File: _maps/map_files/Pahrump/Pahrump-Surface-2.dmm [hunk 2]
```diff
@@ -24336,6 +24376,19 @@
...
+	dynamic_lighting = 1
+	})
```
Comment (MarinaGryphon) [inline _maps/map_files/Pahrump/Pahrump-Surface-2.dmm]:
```suggestion
```

File: _maps/map_files/Pahrump/Pahrump-Surface-2.dmm [hunk 3]
```diff
@@ -24336,6 +24376,19 @@
...
+	dir = 1;
+	pixel_y = 23
+	},
+/turf/open/floor/f13{
+	color = "#a6968e";
+	icon_state = "darkrusty";
+	name = "dirty floor"
+	},
+/area/f13/wasteland{
+	dynamic_lighting = 1
```
Comment (MarinaGryphon) [inline _maps/map_files/Pahrump/Pahrump-Surface-2.dmm]:
```suggestion
```

**Review by MarinaGryphon — CHANGES_REQUESTED** [review]
An area has been varedited on the map, which is bad. I fixed this in an earlier commit, but I think the change got overwritten. Please correct this.

**Comment by OdysiaHallow** [comment]
WHY ARE YOU NOT COMPILING REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Comment by BadIsMyUsername** [comment]
> WHY ARE YOU NOT COMPILING REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Click on the details button(s) of the checks, they provide useful information.

Here's some errors from the Windows Build as an example
>ERROR: Non-TGM formatted map detected. Please convert it using Map Merger!
>_maps/map_files/Pahrump/Pahrump-Surface-2.dmm:	pixel_w = 5;
>_maps/map_files/Pahrump/Pahrump-Surface-2.dmm:	pixel_w = 5;
>ERROR: incorrect pixel offset variables detected in maps, please remove them.

**Pull request closed without merging**